### ISSUE TITLE
Blacklist modded structures?

### ISSUE THREAD

**Issue by SpeidVincent** [MaxenceDC/sparsestructures#15]
Mod page says their was a way to blacklist structures added in v2.0. However, I can't seem to find out how to do so.

**Comment by MaxenceDC** [comment]
Oh, so maybe this is on me, by "blacklisting" a structure, I meant to make it unaffected by the mod, not disabled. If you want to keep a structure unaffected, then set its sparsity factor to 1. Otherwise, if you want to remove every instance of said structure, it's not currently possible with this mod, though I think it's possible with Cristel Lib. This feature might be added in the future.

**Comment by Miamlya** [comment]
Yeah, the ability to disable structures could be a neat feature for certain use cases!

**Comment by MaxenceDC** [comment]
v3.0
